v2/validator.go

```diff
@@ -73,3 +79,29 @@ func ValidateAutoEventFrequency(fl validator.FieldLevel) bool {
 	_, err := time.ParseDuration(fl.Field().String())
 	return err == nil
 }
+
+// ValidateUpdateDtoId used to check the UpdateDTO Id pointer value
+func ValidateUpdateDtoId(fl validator.FieldLevel) bool {
+	val := fl.Field()
+	// Since the Id should be string type, so skip the validation when it isn't a kind of string
+	if val.Kind() != reflect.String {
+		return true
+	}
+	_, err := uuid.Parse(val.String())
+	return err == nil
+}
+
+// ValidateUpdateDtoName used to check the UpdateDTO name pointer value
+func ValidateUpdateDtoName(fl validator.FieldLevel) bool {
```

Comment:
I like this and see that it can be expanded to validate the `name` only have valid characters.

Reply:
ok, we will add the validation for name when resolving https://github.com/edgexfoundry/go-mod-core-contracts/issues/343